CHANGELOG.md

```diff
@@ -54,6 +54,7 @@
 * [ENHANCEMENT] Ingester/Querier: Optimise regexps with long lists of alternates. #8221, #8234
 * [ENHANCEMENT] Ingester: Include more detail in tracing of queries. #8242
 * [EHNAHCEMENT] Distributor: add `insight=true` to remote-write and OTLP write handlers when the HTTP response status code is 4xx. #8294
+* [BUGFIX] Distributor: prometheus retry on 5xx and 429 errors, while otlp collector only retry on 429, 502,503 and 504, mapping other 5xx errors to the retryable ones in otlp endpoint. #8324
```

Comment:
Is it really a bug fix? IMO a bug fix is something that we introduced in a previous release (so prior to Mimir 2.12) and that we want to fix in this release (so in Mimir 2.13). Is it really the case here?
If yes, then it is OK to call it a bugfix. Otherwise it is an enhancement.

[nit]: 429, 502,503 and 504 => 429, 502, 503 and 504

Reply:
It comes from an incident, that user could lost data due to not retry.